deeppavlov/configs/classifiers/yahoo_convers_vs_info.json

```diff
@@ -0,0 +1,174 @@
+{
+  "dataset_iterator": {
```

Comment:
i feel that if there is no reader, there should be no iterator and no `in_y` and `train` blocks

Reply:
yes. removed